templates/main.html

```diff
@@ -209,7 +209,8 @@ <h3>
     </div>
     <footer>
       <small data-i18n="gci-copyright">Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small><br />
-      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span></small>
+      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span></small><br/>
+      <small><span>Built by <a href="https://coala.io/">coala</a>.</span></small>
```

Comment:
Maybe i18nize the span? Something like:

```html
<small><span data-i18n="built-by">Built by</span> <a href="https://coala.io/">coala</a>.</small>
```

Reply:
i thought of adding i18n but thought since  translation strings are not available for other languages, I didnt need it. It can be done but first I want to finish this stupid checkins, where I spent 5 days when I made the actual fix  in 1 hour.